core/src/main/java/com/novoda/merlin/receiver/ConnectivityChangeEvent.java

```diff
@@ -8,6 +8,10 @@
     private final String info;
     private final String reason;
 
+    public ConnectivityChangeEvent(boolean isConnected) {
```

Comment:
Could this be a factory method instead, with a meaningful name? Something like `createNoNetworkInfoInstance()`